forte/data/vocabulary.py

```diff
@@ -116,13 +116,14 @@ def __init__(self, method: str, need_pad: bool, use_unk: bool):
 
     def __getstate__(self):
         r"""For serialization."""
-        state = {}
-        state["method"] = self.method
-        state["need_pad"] = self.need_pad
-        state["use_unk"] = self.use_unk
-        state["element2id_dict"] = self.element2id_dict
-        state["id2element_dict"] = self.id2element_dict
-        state["next_id"] = self.next_id
+        state = {
```

Comment:
Is this function simply return everything of this vocab class into a dict? If so, you don't need to implement __getstate__ and __setstate__ at all. Python's default `__dict__` will be used (https://docs.python.org/3/library/pickle.html#object.__getstate__)

Reply:
That's good to know. Updated.